modules/ROOT/pages/7.1.1-release-notes.adoc

```diff
@@ -59,6 +59,30 @@ For information on the **<Open source plugin name>** plugin, see xref:<plugincod
 
 The following premium plugin updates were released alongside {productname} {release-version}.
 
+=== PowerPaste
+
+The {productname} {release-version} release includes an accompanying release of the **PowerPaste** premium plugin.
+
+**PowerPaste** includes the following fix.
+
+==== List with start attribute separated by `p` tags resulted in wrong numbering of the lists.
```

Comment:
Please improve this heading

Reply:
```suggestion
==== A list with the start attribute, separated by `p` tags, resulted in incorrect numbering of lists.
```

Reply:
@tiny-ben-tran See sugested improvement.